src/quemb/kbe/pbe.py

```diff
@@ -759,15 +760,7 @@ def oneshot(
 
             self.ebe_tot = rets[0]
 
-        if not calc_frag_energy:
```

Comment:
This call to `compute_energy_full` here is removed, because it is not existing

#35 is related